src/pages/guides/using/contentlogs/contentlogs-setup.md

```diff
@@ -0,0 +1,179 @@
+
+## Introduction
+
+Content Logs events provide all details of end user interactions with the assets. This is ideal for enterprises that require automated compliance tracking, security analytics, or integration into existing operational workflows.
+
+It allows real-time ingestion into security and monitoring platforms such as Splunk, Microsoft Sentinel, Sumo Logic, and other SIEM or data analytics tools.
+
+## Setup Adobe I/O
+
+See [Getting Started with Adobe I/O Events](../../../index.md)
+
+For basic instructions for this use case, starting from [developer.adobe.com/console](https://developer.adobe.com/console/):
+
+*When prompted, click the designated button to proceed*
+
+- Select `Create new project`
+
+  ![Create new project](../../img/contentlogs/create_new_project.png "Quick Start")
+
+- Select `Add event`
+
+  ![Add event](../../img/contentlogs/add_event.png "Get started with your new project by adding an event subscription")
+
+- Filter by `Creative Cloud`
+  ![Filter Selection](../../img/contentlogs/filter_creative_cloud.png "Filter by Creative Cloud")
+
+- Select `Select Content Log Events and click Next`
+
+  ![Provider selection](../../img/contentlogs/select_content_logs.png "Select event provider")
+
+- Subscribe to the content log event types of your choosing and click `Next`
+
+  ![Event selection](../../img/contentlogs/choose_interested_event_types.png "Select event subscriptions")
+
+- Set up OAuth Server-to-Server Credentials and click `Next`
+    - The OAuth Server-to-Server credential relies on the OAuth 2.0 client_credentials grant type to generate access tokens.
+
+  ![Set up credentials](../../img/contentlogs/add_server_to_server_oauth.png "Set up credentials")
+
+- Set up Event Registration and click `Next`
+    - Provide a name and description for this event subscription
+
+  ![Set Up registration](../../img/contentlogs/event_reg_details.png "Set Up registration")
+
+- Configure Event Registration and click `Save configured events`
+
+  ![Configure registration](../../img/contentlogs/add_event_notification_mode.png "Configure registration")
+
+    - Journaling is selected by default which is used for Streaming.
+    - Optionally choose whether to enable Webhook or Runtime action
+        - Enable Webhook
+            - We recommend batch over single webhooks
+            - For `Webhook URL` a public https endpoint must be provided
+            - The endpoint must be able to handle get and post requests
+            - The get request must respond with the challenge query if it exists
+            - The post request must respond that it received the message or the webhook will re-attempt to send several times before giving up and automatically disabling the webhook sends
+        - Enable Runtime action
+            - [See Setting up your Runtime Environment](https://developer.adobe.com/runtime/docs/guides/getting-started/)
+            - Select a pre-made runtime action/runtime namespace
```

Comment:
can be re-phrased to 
select a non-web runtime action to process events from your namespace. 
Not sure what you mean by pre-made runtime namespace.